addons/spreadsheet/static/src/chart/odoo_chart/odoo_chart_helpers.js

```diff
@@ -86,3 +89,34 @@ export function onOdooChartItemHover() {
         }
     };
 }
+
+export async function navigateToOdooMenu(menu, actionService, notificationService, newWindow) {
+    if (!menu) {
+        throw new Error(`Cannot find any menu associated with the chart`);
+    }
+    if (!menu.actionID) {
+        notificationService.add(
+            _t(
+                "The menu linked to this chart doesn't have an corresponding action. Please link the chart to another menu."
+            ),
+            { type: "danger" }
+        );
+        return;
+    }
+    await actionService.doAction(menu.actionID, { newWindow });
+}
+
+/**
+ * Check if the even is a middle mouse click or ctrl+click
+ *
+ * ChartJS doesn't receive a click event when the user middle clicks on a chart, so we use the mouseup event instead.
+ *
+ */
+export function isMiddleClick(event) {
```

Comment:
if it's exported and exposed, I would name this to make it clear this is something for chartjs events.
`isChartJsMiddleClick` ?